docs/content/community/overview/index.md

```diff
@@ -3,7 +3,7 @@ type: docs
 title: "Radius Community"
 linkTitle: "Overview"
 description: "Information about the Radius community"
-weight: 100
+weight: 200
```

Comment:
```suggestion
weight: 100
```

Reply:
why is this 100? refrence section index page has a weight of 100